tools/topology/topology1/sof-adl-sdw-cs42l42.m4

```diff
@@ -0,0 +1,286 @@
+#
+# Topology for Tigerlake with rt711 + rt1308 (x2).
+#
+
```

Comment:
why is there a need to create a new topology just to support a new headset codec?

Put differently, why not use the existing topology or describe what's missing?

Topology files are a PITA to maintain, less is more really.

Reply:
Thanks for the suggestions.
These modifications I did is a temproary solution to enable CS42L42 codec in ADL platform.

> Topology files are a PITA to maintain, less is more really.
Agreed with you. :)